client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/translations.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @jest-environment jsdom
+ */
+import { getLaunchpadTranslations } from '../translations';
+
+describe( 'Translations', () => {
+	describe( 'getLaunchpadTranslations', () => {
+		describe( 'when using a tailored onboarding flow', () => {
+			it( 'provides flow specific text', () => {
+				const newsletterTranslations = getLaunchpadTranslations( 'newsletter' );
+				expect( newsletterTranslations.flowName ).toEqual( 'Newsletter' );
+				expect( newsletterTranslations.sidebarTitle ).toEqual(
+					'Your Newsletter is ready to launch!'
+				);
```

Comment:
These tests will ensure that the copies for tailored flows dont change unexpectedly. However, they dont ensure anything about translation (not that id expect them to or for us to test that). I do wonder about the level of value we are adding with these tests.

That said, they do add some value and they are working well on my end.

Reply:
I was mulling this over as well. Rather than test specific, hard-coded text, I wonder if it would be better to just test whether or not something like getLaunchpadTranslations( 'newsletter' ) and getLaunchpadTranslations( null ) return different values.

All we really care about is that _some_ sort of flow specific text is being generated from the function, right? 🤔

Reply:
True, but its not bad that we are checking the specific flows for 'link-in-bio', 'launchpad', etc. still work properly like we are here. More safety!

Reply:
Sounds good 👍 I'll rebase and merge shortly